frontend/src/lib/lemon-ui/icons/icons.tsx

```diff
@@ -2205,3 +2205,11 @@ export function XmasTree(props: LemonIconProps): JSX.Element {
         </LemonIconBase>
     )
 }
+
+export function IconRobot(props: LemonIconProps): JSX.Element {
```

Comment:
Can you add where you got this icon from, for context? I couldn't find this in Material Icons